### ISSUE TITLE
'NoneType' object has no attribute 'telemetry'

### ISSUE THREAD

**Issue by Kurt-von-Laven** [Lucas-C/pre-commit-hooks-safety#29]
The hook at v1.2.4 has been failing for me since 12:14 a.m. UTC today (June 29th, 2022) in one repo but not others. I believe the underlying API is [reported as up](https://status.pyup.io/), and I'm stumped as to how to debug this further, so any help would be greatly appreciated. This is the error I'm getting:

```
Unexpected Exception happened: 'NoneType' object has no attribute 'telemetry'
Traceback (most recent call last):
  File "/home/runner/.cache/pre-commit/reponl89_evy/py_env-python3.10.4/lib/python3.10/site-packages/safety/cli.py", line 101, in check
    announcements = safety.get_announcements(key=key, proxy=proxy_dictionary, telemetry=ctx.parent.telemetry)
AttributeError: 'NoneType' object has no attribute 'telemetry'
```

**Comment by Sriharan-Manogaran** [comment]
Facing the same issue since today morning (2.00 am UTC). No libraries are updated, everything is the same as yesterday.

Python version: 3.8.9
Pre-commit: 4.1.0 (also tried with 4.3.0)
Safety: 1.2.4


```
Unexpected Exception happened: 'NoneType' object has no attribute 'telemetry'
Traceback (most recent call last):
  File "/Users/sriharanm/.cache/pre-commit/repor1y6par0/py_env-python3.8/lib/python3.8/site-packages/safety/cli.py", line 101, in check
    announcements = safety.get_announcements(key=key, proxy=proxy_dictionary, telemetry=ctx.parent.telemetry)
AttributeError: 'NoneType' object has no attribute 'telemetry'
Unhandled exception happened: 'NoneType' object has no attribute 'telemetry'
```

**Comment by sshishov** [comment]
The problem is the underlying library `safety` has been upgraded to `2.0.0` version with breaking changes.
There are 2 ways to fix the issue, one is to fix this version by pinning the requirement.
Another fix would be to start supporting new version of `safety` package.

Both these fixes can be done separately.

**Comment by Sriharan-Manogaran** [comment]
@Lucas-C Is it possible to roll out this soon.

**Comment by va1ha11a** [comment]
Same issue here

**Comment by luizsla** [comment]
Same issue here :(

![safety_error](https://user-images.githubusercontent.com/25124681/176896537-3cc0914f-1451-4da4-9f94-98a4fd96fac7.png)

**Comment by Lucas-C** [comment]
Thank you for the reports

This has been fixed in [release v1.3.0](https://github.com/Lucas-C/pre-commit-hooks-safety/releases/tag/v1.3.0)
